<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>百度图库采集器</string>
  </property>
  <widget class="QLabel" name="keywordLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入图片关键字：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="keywordLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="numLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置爬取数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="savePathLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置保存路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePathBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片保存路径</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停止采集</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
